app/src/main/java/com/simplecity/amp_library/model/Genre.java

```diff
@@ -57,10 +57,7 @@ public boolean equals(Object o) {
 
         Genre genre = (Genre) o;
 
-        if (id != genre.id) return false;
-        if (numSongs != genre.numSongs) return false;
-        return name != null ? name.equals(genre.name) : genre.name == null;
-
+        return id == genre.id && numSongs == genre.numSongs && (name != null ? name.equals(genre.name) : genre.name == null);
```

Comment:
I'd prefer not to make this change. 'Simplifying' the conditional statement generally makes it harder to read for equals() method.